inclusion_connect/accounts/forms.py

```diff
@@ -160,11 +160,26 @@ def __init__(self, *args, initial, **kwargs):
         email_field.widget.attrs = EMAIL_FIELDS_WIDGET_ATTRS.copy()
         email_field.disabled = "email" in initial
 
+    def send_mail(
+        self,
+        subject_template_name,
+        email_template_name,
+        context,
+        from_email,
+        to_email,
+        html_email_template_name=None,
+    ):
+        super().send_mail(
+            subject_template_name="registration/password_reset_subject.txt",
+            email_template_name="registration/password_reset_email.txt",
+            context=context,
+            from_email=from_email,
+            to_email=to_email,
+            html_email_template_name="registration/password_reset_email.html",
```

Comment:
Ça permet de les redéfinir au besoin, au lieu de les jeter silencieusement.

```suggestion
        html_email_template_name="registration/password_reset_email.html",
    ):
        super().send_mail(
            subject_template_name=subject_template_name,
            email_template_name=email_template_name,
            context=context,
            from_email=from_email,
            to_email=to_email,
            html_email_template_name=html_email_template_name,
```